.github/workflows/build.yaml

```diff
@@ -7,22 +7,52 @@ name: Buid iOS app
 on:
   workflow_call:
 
+env:
+  spm_dir: './SPM'
+  cocoapods_dir: './CocoaPods'
+  platform: 'iOS Simulator'
+  device_name: 'iPhone 14'
+  ios_version: '16.2'
+
 # clear out default permissions
 permissions: {}
 
 jobs:
-  build:
-    name: Buid iOS app
+  build-spm:
```

Comment:
Doesn't need to be done now, but it would be a great idea to create a template for these build jobs. From what i see, we could even make it generic enough that we would just have one template for both spm and cocoapods use cases. The only difference from what i see is the `working-directory`. It would also allow us to use that template when we add new examples.

Reply:
As you mentioned, having more examples will make it necessary to do so.